docs/administrator-documentation/moderne-platform/references/reporting.md

```diff
@@ -23,7 +26,7 @@ Audit logs are accessible from `https://<TENANT>.moderne.io/admin/audit-logs`, a
 
 To download audit logs, use the "Export to CEF" button: ![](./assets/audit-log-download.png)
 
-## Other reports (recipe runs and commits)
+## Other reports (recipe runs, commits and usage)
 
 To access non-audit-log reports, navigate to `https://<TENANT>.moderne.io/admin/reports`.
```

Comment:
The update images in this document looks like the below

<img width="1090" alt="Screenshot 2024-12-10 at 3 18 43 PM" src="https://github.com/user-attachments/assets/eb25a073-29d7-4e7d-896d-d8d84f194dfc">